### DISCUSSION TITLE
Legacy BIOS avoids Ventoy installed disk

### DISCUSSION THREAD

**Discussion by ghost** [ventoy/Ventoy#2798] [Q&A]
This is a problem that i have from the beginning i tested all of the other factors (other usb's, other iso's, x64/x32, mbr/gpt, secure boot enabled/disabled, and many..) since this program released but nothing worked. My Legacy BIOS just avoids booting from Ventoy Disk and skips to booting Windows directly even tho i select exact Ventoy Disk.

MY PC SPECS:
HP DC7800P SFF (Well Upgraded)
GT 710 1 GB GDDR5
6 GB DDR2 667 Mhz
Q9400
240 GB SSD
SYSTEM IS ALLY BASED ON MBR AND USING LEGACY BOOT SYSTEM

I dont know what can i share more to explain myself to you guys, please just ask me and i'll reply detailed beassured.

**Comment by ghost** [comment]
Whenever i try to boot from it usually just stays a blinking cursor that keep blinking on the top left forever too only option to make is ctrl+alt+del to reboot pc and same happens again.

**Comment by steve6375** [comment]
Can the Ventoy usb drive boot on other systems?
Can you boot using QEMU?
Try easy2boot.xyz, it includes QEMU test program in root of usb drive 
How did you make the Ventoy drive?
What usb drives have you tried (good make, expensive or cheap 2tb fake?)
